### ISSUE TITLE
Issues with Exploration Skill and Better Cartography Table

### ISSUE THREAD

**Issue by TheCavemanOfficial** [nbusseneau/BetterCartographyTable#11]
Im not entirely sure which one is causing the issue, but when i attempt to write to the cartography table it just opens the map, it refuses to write, i have an exploration skill level of 68 (40 to write on the table) and i cannot write my map to the table. Holding shift does not help.

**Comment by nbusseneau** [comment]
Indeed, looking at how Exploration does its thing, I am not surprised the mods are not compatible. Maybe I can add a compatibility layer such that BCT detects if Exploration is installed, and if yes uses its configuration to allow/disallow things.

I will think on this, not promising anything however, it might be that after investigating I find it too bothersome to implement 😅
